src/mesido/heat_physics_mixin.py

```diff
@@ -3218,7 +3303,9 @@ class and the selected temperature in the network.
     def __ates_max_stored_heat_constriants(self, ensemble_member):
         constraints = []
 
-        for ates in self.energy_system_components.get("ates", []):
+        for ates in [
+            *self.energy_system_components.get("ates", []),
+        ]:
```

Comment:
depends on how we make the WKO model. I left it out as this is now specific for the new HT-ATES model. If we can get the same functionality from setting bounds in the WKO model it would be preferable.

Reply:
agree